src/main/java/org/folio/consortia/exception/UserTenantNotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package org.folio.consortia.exception;
+
+public class UserTenantNotFoundException extends ResourceNotFoundException {
```

Comment:
I think its over engineering, it's enough only one NotFound exception for all entities

Reply:
I think it needs only specify which object not found
![image](https://user-images.githubusercontent.com/113523904/224480238-bd8dc2a4-d6a9-4811-b307-2f026604a29e.png)

Reply:
If we remove error message be more general like Object with id [120120-a] not found instead of User Tenant with id [23123] not found. Can I remove in this case?

Reply:
you can provide message param and thow an exception with any message provided